### ISSUE TITLE
competition_wcif_spec seems to only pass if you run it in UTC time

### ISSUE THREAD

**Issue by jfly** [thewca/worldcubeassociation.org#3574]
Here's what I see when running `bin/rspec spec/models/competition_wcif_spec.rb` locally:

```
  1) Competition WCIF #to_wcif renders a valid WCIF
     Failure/Error:
       expect(competition.to_wcif).to eq(
         "formatVersion" => "1.0",
         "id" => "TestComp2014",
         "name" => "Test Comp 2014",
         "shortName" => "Test 2014",
         "persons" => [delegate.to_wcif(competition)],
         "events" => [
           {
             "id" => "333",
             "rounds" => [
     
       expected: {"events"=>[{"id"=>"333", "rounds"=>[{"advancementCondition"=>{"level"=>16, "type"=>"ranking"}, "cuto...", "id"=>3, "name"=>"Room 2 for venue 2"}], "timezone"=>"Europe/Paris"}]}, "shortName"=>"Test 2014"}
            got: {"events"=>[{"id"=>"333", "rounds"=>[{"advancementCondition"=>{"level"=>16, "type"=>"ranking"}, "cuto...", "id"=>3, "name"=>"Room 2 for venue 2"}], "timezone"=>"Europe/Paris"}]}, "shortName"=>"Test 2014"}
     
       (compared using ==)
     
       Diff:
       @@ -3,6 +3,6 @@
        "id" => "TestComp2014",
        "name" => "Test Comp 2014",
        "persons" => [{"avatar"=>{"thumbUrl"=>"http://test.host/assets/missing_avatar_thumb-f0ea801c804765a22892b57636af829edbef25260a65d90aaffbd7873bde74fc.png", "url"=>"http://test.host/assets/missing_avatar_thumb-f0ea801c804765a22892b57636af829edbef25260a65d90aaffbd7873bde74fc.png"}, "birthdate"=>"1980-01-01", "countryIso2"=>"HU", "email"=>"ianswift@macgyver.org", "gender"=>"m", "name"=>"Donnie Osinski PhD", "personalBests"=>[], "registrantId"=>nil, "registration"=>nil, "roles"=>["delegate"], "wcaId"=>"2016DYZM01", "wcaUserId"=>56}],
       -"schedule" => {"numberOfDays"=>3, "startDate"=>"2014-02-03", "venues"=>[{"id"=>1, "latitudeMicrodegrees"=>123456, "longitudeMicrodegrees"=>123456, "name"=>"Venue 1", "rooms"=>[{"activities"=>[{"activityCode"=>"other-lunch", "childActivities"=>[], "endTime"=>"2014-02-03T13:00:00Z", "id"=>1, "name"=>"Some name", "startTime"=>"2014-02-03T12:00:00Z"}, {"activityCode"=>"333fm-r1", "childActivities"=>[{"activityCode"=>"333fm-r1-g1", "childActivities"=>[], "endTime"=>"2014-02-05T10:30:00Z", "id"=>3, "name"=>"first group", "startTime"=>"2014-02-05T10:00:00Z"}, {"activityCode"=>"333fm-r1-g2", "childActivities"=>[{"activityCode"=>"333fm-r1-g2-a1", "childActivities"=>[], "endTime"=>"2014-02-05T11:00:00Z", "id"=>5, "name"=>"some nested thing", "startTime"=>"2014-02-05T10:30:00Z"}], "endTime"=>"2014-02-05T11:00:00Z", "id"=>4, "name"=>"second group", "startTime"=>"2014-02-05T10:30:00Z"}], "endTime"=>"2014-02-05T11:00:00Z", "id"=>2, "name"=>"another activity", "startTime"=>"2014-02-05T10:00:00Z"}], "color"=>"#304a96", "id"=>1, "name"=>"Room 1 for venue 1"}], "timezone"=>"Europe/Paris"}, {"id"=>2, "latitudeMicrodegrees"=>123456, "longitudeMicrodegrees"=>123456, "name"=>"Venue 2", "rooms"=>[{"activities"=>[], "color"=>"#304a96", "id"=>2, "name"=>"Room 1 for venue 2"}, {"activities"=>[], "color"=>"#304a96", "id"=>3, "name"=>"Room 2 for venue 2"}], "timezone"=>"Europe/Paris"}]},
       +"schedule" => {"numberOfDays"=>3, "startDate"=>"2014-02-03", "venues"=>[{"id"=>1, "latitudeMicrodegrees"=>123456, "longitudeMicrodegrees"=>123456, "name"=>"Venue 1", "rooms"=>[{"activities"=>[{"activityCode"=>"other-lunch", "childActivities"=>[], "endTime"=>"2014-02-03T21:00:00Z", "id"=>1, "name"=>"Some name", "startTime"=>"2014-02-03T20:00:00Z"}, {"activityCode"=>"333fm-r1", "childActivities"=>[{"activityCode"=>"333fm-r1-g1", "childActivities"=>[], "endTime"=>"2014-02-05T18:30:00Z", "id"=>3, "name"=>"first group", "startTime"=>"2014-02-05T18:00:00Z"}, {"activityCode"=>"333fm-r1-g2", "childActivities"=>[{"activityCode"=>"333fm-r1-g2-a1", "childActivities"=>[], "endTime"=>"2014-02-05T19:00:00Z", "id"=>5, "name"=>"some nested thing", "startTime"=>"2014-02-05T18:30:00Z"}], "endTime"=>"2014-02-05T19:00:00Z", "id"=>4, "name"=>"second group", "startTime"=>"2014-02-05T18:30:00Z"}], "endTime"=>"2014-02-05T19:00:00Z", "id"=>2, "name"=>"another activity", "startTime"=>"2014-02-05T18:00:00Z"}], "color"=>"#304a96", "id"=>1, "name"=>"Room 1 for venue 1"}], "timezone"=>"Europe/Paris"}, {"id"=>2, "latitudeMicrodegrees"=>123456, "longitudeMicrodegrees"=>123456, "name"=>"Venue 2", "rooms"=>[{"activities"=>[], "color"=>"#304a96", "id"=>2, "name"=>"Room 1 for venue 2"}, {"activities"=>[], "color"=>"#304a96", "id"=>3, "name"=>"Room 2 for venue 2"}], "timezone"=>"Europe/Paris"}]},
        "shortName" => "Test 2014",
       
     # ./spec/models/competition_wcif_spec.rb:36:in `block (3 levels) in <top (required)>'
     # -e:1:in `<main>'

Finished in 13.12 seconds (files took 0.48921 seconds to load)
28 examples, 1 failure

Failed examples:

rspec ./spec/models/competition_wcif_spec.rb:35 # Competition WCIF #to_wcif renders a valid WCIF

```

It's hard to see, but the diff above is specifically in timestamps, so this feels like a timezone bug.

**Comment by jfly** [comment]
(discovered while paring with @danayi13)

**Comment by viroulep** [comment]
Ah nice find!
I've fixed in #3207, since I ran into it locally.
